modin/pandas/test/test_series.py

```diff
@@ -2271,12 +2271,11 @@ def test_reorder_levels():
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 @pytest.mark.parametrize(
-    "repeats", [2, 3, 4], ids=["repeats_{}".format(i) for i in [2, 3, 4]]
+    "repeats", [0, 2, 3, 4], ids=["repeats_{}".format(i) for i in [0, 2, 3, 4]]
```

Comment:
We should also add a test for a list of integers as `repeats`.

Reply:
added in the last commit